post/v1/service.proto

```diff
@@ -8,7 +8,7 @@ message OperatorStatusRequest {}
 
 message OperatorStatusResponse {
   enum Status {
-    _UNUSED = 0;
+    UNUSED = 0;
```

Comment:
Why change it?

Reply:
`_UNUSED` violates the linting rule `ENUM_VALUE_UPPER_SNAKE_CASE`